lib/ash/code_interface.ex

```diff
@@ -832,6 +832,7 @@ defmodule Ash.CodeInterface do
                             bulk_opts
                             |> Keyword.put(:return_records?, true)
                             |> Keyword.put(:return_errors?, true)
+                            |> Keyword.put(:return_notifications?, true)
```

Comment:
I think the missing piece is `notify?: true` here, and then the rest can likely be omitted?